### PR TITLE
feat(alerting): add email sender

### PR DESCRIPTION
**Pull request by iamKunalGupta**
New ENV Vars:
- `PEERDB_ALERTING_EMAIL_SENDER_SOURCE_EMAIL`
- `PEERDB_ALERTING_EMAIL_SENDER_CONFIGURATION_SET`
- `PEERDB_ALERTING_EMAIL_SENDER_REGION`
- `PEERDB_ALERTING_EMAIL_SENDER_REPLY_TO_ADDRESSES`

- [x] test aws cross region sending
  - config set is per region, have added env for setting a specific region

UI:
image

### CODE REVIEW

File: flow/alerting/slack_alert_sender.go [hunk 1]
```diff
@@ -7,34 +7,46 @@ import (
 	"github.com/slack-go/slack"
 )
 
-type slackAlertSender struct {
+type SlackAlertSender interface {
+	AlertSender
+}
+
+type slackAlertSenderImpl struct {
```
Comment (serprex) [inline flow/alerting/slack_alert_sender.go:14]:
```suggestion
type SlackAlertSender struct {
```
Don't need that much abstraction

File: flow/alerting/email_alert_sender.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	"github.com/aws/aws-sdk-go-v2/service/ses/types"
+
+	"github.com/PeerDB-io/peer-flow/peerdbenv"
+)
+
+type EmailAlertSender interface {
+	AlertSender
+}
+
+type emailAlertSenderImpl struct {
```
Comment (serprex) [inline flow/alerting/email_alert_sender.go:18]:
```suggestion
type EmailAlertSender struct {
```
See comment in slack alerter

File: flow/alerting/email_alert_sender.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+func newSesClient(ctx context.Context, region *string) (*ses.Client, error) {
+	sdkConfig, err := config.LoadDefaultConfig(ctx, func(options *config.LoadOptions) error {
+		if region != nil {
+			options.Region = *region
+		}
+		return nil
+	})
+	if err != nil {
+		return nil, err
+	}
```
Comment (serprex) [inline flow/alerting/email_alert_sender.go:97]: This looks like it can be shared with SNS code

Comment (iamKunalGupta) [reply flow/alerting/email_alert_sender.go:97]: Will test the SES code once and will abstract away then

File: ui/app/alert-config/page.tsx [hunk 4]
```diff
@@ -141,7 +158,8 @@ const AlertConfigPage: React.FC = () => {
           {inEditOrAddMode ? 'Cancel' : 'Add Configuration'}
         </Label>
       </Button>
-      {inEditOrAddMode && <NewAlertConfig {...editAlertConfig} />}
+      {/*{inEditOrAddMode && <NewAlertConfig {...editAlertConfig} />}*/}
```
Comment (serprex) [inline ui/app/alert-config/page.tsx:161]:
```suggestion
```

File: flow/alerting/email_alert_sender.go (hunk 3 above)
Comment (iamKunalGupta) [reply flow/alerting/email_alert_sender.go]: done

**Review by Amogh-Bharadwaj — APPROVED** [review]

**Pull request merged**